### ISSUE TITLE
Stores Page Design

### ISSUE THREAD

**Issue by nathanhollows** [nathanhollows/NuggetNav#11]
I would like to implement something like this on the stores page.

Here's what needs to be visible:

- [x] Store name
- [x] Overall rating
- [ ] Average ratings
- [x] Pricing
- [x] Reviews
- [ ] Images
- [x] Write a review #3 (This view flows into that)

![DeepinScreenshot_select-area_20191026141707](https://user-images.githubusercontent.com/13064427/67612035-95960680-f7fb-11e9-9be3-6b7c5bf024f4.png)
![DeepinScreenshot_select-area_20191026141722](https://user-images.githubusercontent.com/13064427/67612036-96c73380-f7fb-11e9-9a25-fa9c1a282856.png)

**Comment by markuz449** [comment]
Will the images be taken from the reviews or do you want new images of the store? Also if there are no images should the images header be displayed?

**Comment by nathanhollows** [comment]
Images should be taken from the reviews, and that can be supplied as it's own API call. If there are no images then the section should not be shown. Same with the prices section

**Comment by nathanhollows** [comment]
Also, I think it would look better if reviews only showed the first three lines and had the option to tap for more

**Comment by markuz449** [comment]
Yeah I agree (p.s check your phone, ill remove this after you have)
